src/app/rmm/me.ts

```diff
@@ -27,8 +28,8 @@ export class Me {
         public app: RMM,
     ) {
     }
-    load() {
-        if ( this.data ) { return; }
+    load(): Observable<any> {
+        if ( this.data || this.is_busy ) { return; }
```

Comment:
Isn't this bare `return;` violating the type constraint?

Reply:
what should it return